targets/zephyr/src/Makefile

```diff
@@ -19,4 +19,4 @@ endif
 # Adding path for jerry script APIs
 ZEPHYRINCLUDE += -I$(JERRY_INCLUDE)
 
-obj-y += main-zephyr.o
+obj-y += main-zephyr.o zephyr_getline.o
```

Comment:
Nitpicking: inconsistent naming scheme of files... dash or underscore? starts with zephyr or ends with zephyr?

(Note: the third filename in the src dir is also somewhat misleading, jerry-entry.c contains only i/o port api implementation.)